test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Alias.Tests.ps1

```diff
@@ -120,39 +135,39 @@ Describe "Export-Alias DRT Unit Tests" -Tags "CI" {
 		{
 			chmod 777 $fulltestpath
 		}
-		
+
     }
 }
 
 Describe "Export-Alias" -Tags "CI" {
-    $testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ExportAliasTestDirectory
-    $testAliases        = "TestAliases"
-    $fulltestpath       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+
+	BeforeAll {
+		$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ExportAliasTestDirectory
+		$testAliases        = "TestAliases"
+		$fulltestpath       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
+	}
 
     BeforeEach {
-	New-Item -Path $testAliasDirectory -ItemType Directory -Force
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force
     }

```

Comment:
should probably add cleanup in an `AfterEach` as it looks like we're creating each time (and `new-item -force` doesn't remove the directory if it exists)